### DISCUSSION TITLE
Specific part rendering not working in razor view.

### DISCUSSION THREAD

**Discussion by giannik** [OrchardCMS/OrchardCore#15895] [Q&A]
Following the example here 
https://orchardcore.readthedocs.io/en/latest/OrchardCore.Modules/OrchardCore.Themes/README/
 
 I have created specific razor template for Project content type  Content-Project.cshtml
 and addded
 
`@await DisplayAsync(Model.Content.MarkdownBodyPart)`

This does not render the part in razor.

The same example in liquid works.

Content-Project.liquid
`  {{ Model.Content.MarkdownBodyPart | shape_render }}`

I prefer to us the razor method but this is blocking me.

**Comment by remesq** [comment]
@giannik Were you able to overcome this somehow?

**Comment by hishamco** [comment]
I'm not sure why you didn't render the markdown as HTML?!!

**Comment by sebastienros** [comment]
@hishamco it is recommended to use the part rendering logic from the driver as it will apply all the options for instance. Use the Markdown directly if you don't care about the part's behavior, just its content. So it's totally ok to do what @giannik is trying to do.

**Comment by sebastienros** [comment]
Code looks good. Can someone confirm by creating a Project type with a MarkdownPart ?
